### ISSUE TITLE
Two different AuthenticationProperties types

### ISSUE THREAD

**Issue by khellang** [aspnet/HttpAbstractions#966]
Why is there two different `AuthenticationProperties` types in this repository?

- [Microsoft.AspNetCore.Http.Authentication](https://github.com/aspnet/HttpAbstractions/blob/3e3772eecd4cc57399c28a3f899e6b0406ef2e1b/src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticationProperties.cs)
- [Microsoft.AspNetCore.Authentication](https://github.com/aspnet/HttpAbstractions/blob/3e3772eecd4cc57399c28a3f899e6b0406ef2e1b/src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationProperties.cs)

Why doesn't `Microsoft.AspNetCore.Authentication.Abstractions` use the one from `Microsoft.AspNetCore.Http.Abstractions`?

**Comment by davidfowl** [comment]
My guess is that the type in Microsoft.AspNetCore.Http.Authentication is deprecated.

@HaoK ?

**Comment by Tratcher** [comment]
Everything in Http.Authentication was depreciated and moved in 2.0.

**Comment by khellang** [comment]
> Everything in Http.Authentication was depreciated and/or moved in 2.0.

Except they're not. Only `AuthenticationManager` is deprecated.

**Comment by davidfowl** [comment]
@khellang Only AuthenticationManager has the Obsolete attribute. Those types are considered deprecated. I'm not sure why we didn't mark them as such.

**Comment by khellang** [comment]
> Those types are considered deprecated. I'm not sure why we didn't mark them as such.

Right, but unfortunately, there's no tooling that can read the ASP.NET team's minds :wink:

It's kinda confusing when you end up importing the wrong namespace/type and there's nothing telling you that it's obsolete 😝

**Comment by HaoK** [comment]
We settled on obsoleting all the main entry points: (https://github.com/aspnet/HttpAbstractions/pull/863#issuecomment-306851013).  

So there should be no way to actually use the old AuthenticationProperties, but yeah there would be no warning if you were only using the properties type.  But you would get a compile warning/error as soon as you tried to actually do anything with it.

**Comment by poke** [comment]
I originally pointed this out on chat; thanks for opening an issue about it @khellang.

The reason why I encountered this is that while migrating to 2.0, I still had a using on `Microsoft.AspNetCore.Http.Authentication` inside my code. So when I created `AuthenticationProperties` somewhere, I got an error about the ambiguous type which I was not able to resolve at that time without actually knowing which type is used and which isn’t (I only was able to do so later when that object was passed to `SignOutAsync`).

If we still want to keep the obsolete `AuthenticationManager` around for now, maybe we could at least change the `AuthenticationProperties` type it uses to the new type, so that we simply no longer have two identical types in the framework?
